subprojects/chrome-trace/src/main/java/org/gradle/trace/GradleTracingPlugin.java

```diff
@@ -61,8 +61,16 @@ private long getStartTime() {
             try {
                 return buildRequestMetaData.getStartTime();
             } catch (NoSuchMethodError e) {
-                return buildRequestMetaData.getBuildTimeClock().getStartTime();
+                return (long) call(call(buildRequestMetaData, "getBuildTimeClock"), "getStartTime");
             }
         }
     }
+
+    private Object call(Object object, String method) {
```

Comment:
Make this `static` and rename it to `invokeGetter`?

Reply:
I moved the method to a utility class.